test/errors/valueAsTypeExpr-brief.good

```diff
@@ -0,0 +1,2 @@
+error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelDomain.chpl:373: Cannot resolve call: no matching candidates
```

Comment:
Another internal module error and path.  But also, what is causing this?  It doesn't appear to be anything that I can see in the user's source code...?

Reply:
I am not sure what's causing this; I will investigate.